src/main/java/com/akrauze/buscompany/mappers/TripMapper.java

```diff
@@ -11,7 +11,8 @@
 public interface TripMapper {
     TripMapper INSTANCE = Mappers.getMapper(TripMapper.class);
 
-    Trip dtoScheduleToModel(AddTripDtoRequest tripDtoRequest);
+
+    Trip mapToTrip(AddTripDtoRequest tripDtoRequest);
```

Comment:
// Better name than it was. you are know which entity you are mapping, so you need only target name
    // but also you can use mapFrom -- method name  because we already know that it is TripMapper